frontend/assets/javascripts/src/modules/analytics/cmp.es6

```diff
@@ -34,4 +36,9 @@ const checkCCPA = () => new Promise((resolve) => {
 
 const registerCallbackOnConsentChange = (fn) => onConsentChange(fn);
 
-export { getConsentForVendors, checkAllTCFv2PurposesAreOptedIn, checkCCPA, registerCallbackOnConsentChange };
+const createPrivacySettingsLink = () => {
+    const privacySettingsButton = document.querySelector(PRIVACY_SETTINGS_SELECTOR);
+    privacySettingsButton.addEventListener('click', cmp.showPrivacyManager);
```

Comment:
Can we wrap the `addEventListener` call in a conditional `if (privacySettingsButton) { ... }`? Just in case someone removes the link from the template, or updates the class names...

Reply:
👍 Done!